extensions/data-plane/data-plane-spi/src/main/java/org/eclipse/dataspaceconnector/dataplane/spi/manager/DataPlaneManager.java

```diff
@@ -51,4 +52,6 @@
      * Provides transfer state.
      */
     State transferState(String processId);
+
+    void registerTransferService(TransferService transferService);
```

Comment:
nit: missing javadoc. Although the name is clear enough but highlighting it as a `nit ` in case you have some javadoc to add here.